schemas/object_hashes.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Should this be a separate document? Could it be a sub-object inside other documents?
I'm just generally wary of separating out documents too much, because it's going to make the queries complicated

Reply:
I think we should talk to Shane about that one. I think the query that this structure neetly provides is given an object, list the identical objects. I don't know Arango well enough to know if that is better supported by an indexed field.